examples/example.js

```diff
@@ -1,158 +1,216 @@
+const client = new apProto.Aggregator(
+  config[env].AP_AVS_RPC,
+  grpc.credentials.createInsecure()
+);
 
 function asyncRPC(client, method, request, metadata) {
-    return new Promise((resolve, reject) => {
-        client[method].bind(client)(request, metadata, (error, response) => {
-            if (error) {
-                reject(error)
-            } else {
-                resolve(response)
-            }
-        })
-    })
+  return new Promise((resolve, reject) => {
+    client[method].bind(client)(request, metadata, (error, response) => {
+      if (error) {
+        reject(error);
+      } else {
+        resolve(response);
+      }
+    });
+  });
 }
 
 async function signMessageWithEthers(wallet, message) {
-    const signature = await wallet.signMessage(message)
-    return signature
+  const signature = await wallet.signMessage(message);
+  return signature;
 }
 
 async function generateApiToken() {
   // When running from frontend, we will ask user to sign this through their
   // wallet providr
-  const wallet = new Wallet(privateKey)
-  const owner = wallet.address
-  const expired_at = Math.floor(+new Date() / 3600 * 24)
-  const message = `key request for ${wallet.address} expired at ${expired_at}`
-  let signature = await signMessageWithEthers(wallet, message)
-  // console.log(`message: ${message}\nsignature: ${signature}`)
-  let result = await asyncRPC(client, 'GetKey', {
+  const wallet = new Wallet(privateKey);
+  const owner = wallet.address;
+  const expired_at = Math.floor((+new Date() / 3600) * 24);
+  const message = `key request for ${wallet.address} expired at ${expired_at}`;
+  const signature = await signMessageWithEthers(wallet, message);
+  //console.log(`message: ${message}\nsignature: ${signature}`)
+
+  let result = await asyncRPC(
+    client,
+    "GetKey",
+    {
       owner,
       expired_at,
-      signature
-  }, {})
+      signature,
+    },
+    {}
+  );
 
-  return { owner, token: result.key }
+  return { owner, token: result.key };
 }
 
-
 async function listTask(owner, token) {
-  const metadata = new grpc.Metadata()
-  metadata.add('authkey', token)
+  const metadata = new grpc.Metadata();
+  metadata.add("authkey", token);
 
-  const result = await asyncRPC(client, 'ListTasks', {}, metadata)
+  const result = await asyncRPC(client, "ListTasks", {}, metadata);
 
-  console.log("Task ID for ", owner,"\n", result)
+  console.log("Tasks that has created by", owner, "\n", result);
 }
 
 async function getTask(owner, token, taskId) {
-  const metadata = new grpc.Metadata()
-  metadata.add('authkey', token)
+  const metadata = new grpc.Metadata();
+  metadata.add("authkey", token);
 
-  const result = await asyncRPC(client, 'GetTask', { bytes: taskId }, metadata)
+  const result = await asyncRPC(client, "GetTask", { bytes: taskId }, metadata);
 
-  console.log("Inspect TaskID: ".padEnd(30, " "), taskId, "\n")
   result.nodes.filter(e => e != null).map(node => {
```

Comment:
TODO: It would make sense to show the task condition and task trigger type in the get response.

For example, we couldn’t tell the condition in the below get response.
![CleanShot 2024-10-22 at 19 41 13](https://github.com/user-attachments/assets/84d70665-bbca-4b54-9901-8ddb0eb68252)

Originally, they are 
```
Task condition: 
      bigCmp(
        priceChainlink("0x0ee7f0f7796Bd98c0E68107c42b21F5B7C13bcA9"),
        toBigInt("10000")
      ) > 0
Trigger type 2
```